src/core/src/main/java/org/apache/jmeter/services/FileServer.java

```diff
@@ -472,7 +472,7 @@ private BufferedWriter createBufferedWriter(FileEntry fileEntry) throws IOExcept
 
     public synchronized void closeFiles() throws IOException {
         for (Map.Entry<String, FileEntry> me : files.entrySet()) {
-            closeFile(me.getKey(),me.getValue() );
+            closeFile(me.getKey(), me.getValue() );
```

Comment:
This looks like `chaotic neutral` to me: the whitespace before the closing parenthesis looks odd, especially since there was no whitespace after the opening one.